draft-ietf-taps-interface.md

```diff
@@ -1323,8 +1323,8 @@ or categories of interfaces it wants to `Require`, `Prohibit`, `Prefer`, or
 selection to that single interface, and often leads to less flexible and resilient
 connection establishment.
 
-In contrast to other Selection Properties, this property is a tuple of an
-(Enumerated) interface identifier and a preference, and can either be
+In contrast to other Selection Properties, this property is a set of
+(Enumerated) interface identifier and preference tuples, and can either be
```

Comment:
This took me three times reading it… 
```suggestion
In contrast to other Selection Properties, this property is a set of tuples,
where each tuple consists if an (Enumerated) interface identifier and preference tuples.
It and can either be
```

Reply:
Thanks for this suggestion. I agree that this is better - I'll do it in a separate PR.